v0/destinations/pipedrive/data/PipedriveGroup.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "destKey": "name",
+    "sourceKeys": ["traits.name", "context.traits.name"],
```

Comment:
for group calls traits.name contains the name of the group/organisation. 
while context.traits.name contains the name of the user. (persisted traits) 
so we should only map traits.name

Reply:
similar for other propertied in this json too. (add_time etc)